static/js/services/mark-read.js

```diff
@@ -1,37 +1,20 @@
-var _ = require('underscore');
-
 angular.module('inboxServices').factory('MarkRead', function(
   $q,
-  DB,
-  Session
+  DB
 ) {
 
   'use strict';
   'ngInject';
 
-  var updateDoc = function(user, doc) {
-    var readers = doc.read || [];
-    var index = readers.indexOf(user);
-    if (index !== -1) {
-      // already marked as read
-      return;
-    }
-    readers.push(user);
-    doc.read = readers;
-    return doc;
-  };
-
-  var updateDocs = function(user, docs) {
-    return _.compact(_.map(docs, _.partial(updateDoc, user)));
-  };
-
   return function(docs) {
-    var user = Session.userCtx().name;
-    var updated = updateDocs(user, docs);
-    // Conflicts will fail silently. That's ok.
-    if (!updated.length) {
+    if (!docs || !docs.length) {
       return $q.resolve();
     }
-    return DB().bulkDocs(updated);
+    var metaDocs = docs.map(function(doc) {
+      var type = doc.form ? 'report' : 'message';
+      var id = [ 'read', type, doc._id ].join(':');
```

Comment:
So I feel like I'm seeing the definition of the read doc in a few places, and the definition of the meta DB in a few places. Do you think it's possible to standardise that in one place / utils file?